src/SprykerEco/Zed/PunchoutCatalogs/Communication/Plugin/PunchoutCatalogConnectionFormatPluginInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Plugin;
+
+use Spryker\Zed\Kernel\Communication\Form\FormTypeInterface;
+
+interface PunchoutCatalogConnectionFormatPluginInterface extends FormTypeInterface
+{
+    /**
+     * Specification:
+     * - Retrieves the format of the connection.
```

Comment:
`- Retrieves the connection format as a lowercased glossary key.`

Reply:
How it's related to the glossary?

Btw it's specs from https://spryker.atlassian.net/wiki/spaces/CORE/pages/931495983/PS-8578+Punch+Out, so I would keep them for AA review